configs/cms-xcache/xrootd/Authfile-auth

```diff
@@ -0,0 +1,4 @@
+t readcmsdata /store/ lr
+u cmsprod readcmsdata
+u cmsphedex readcmsdata
+u cmsuser readcmsdata
```

Comment:
Please rename the file so it's more clearly cms-xcache related. You'll also need to add a line in the `%files -n cms-xcache` section for this file.